ddht/tools/web3.py

```diff
@@ -343,13 +362,36 @@ class DiscoveryV5Module(ModuleV2):  # type: ignore
         result_formatters=lambda method: find_nodes_response_formatter,
         mungers=[find_nodes_munger],
     )
-    send_find_nodes = Method(
+    send_find_nodes: Method[
```

Comment:
Any idea why mypy was missing these types? They still aren't being correctly validated btw...

Reply:
If you're saying that these type hints aren't actually being validated (i.e. that you could changed one of them arbitrarily and mypy wouldn't complain) then I think that's something we should investigate (probably separately from this PR).  I'd recommend starting by using some `reveal_type` statements to figure out what mypy thinks these types are.

My guess is that web3.py might be leaking `Any` which is typically where this type of deficiency shows up.